src/artifact.c

```diff
@@ -5425,6 +5429,13 @@ boolean printmessages; /* print generic elemental damage messages */
 		if (Hallucination) You("flame the nasty troll!");  //trollsbane hits monsters that pop in to ruin your day.
 		*truedmgptr += d(2, 20) + 2 * otmp->spe; //boosts power better than demonbane hitting silver hating.
 	}
+	
+	if (youagr && is_alien(pd) && oartifact == ART_FALLINGSTAR_MANDIBLES)
+	{
+		*truedmgptr += basedmg;
```

Comment:
Is this a pattern any other artis follow? Is this damage described in the inspection menu for the weapon?

Reply:
yeah, this is the section for trollsbane extra damage, giantslayer slow, plague explosions, etc. it does NOT show up in arti inspect unfortunately, but I could/will probably add a manual line to that display that says as much. trollsbane et. al. should probably get that line added too though :/

Reply:
added a line in invent.c to reflect that, see new commit